controllers/controllers/workloads/cforg_controller_test.go

```diff
@@ -0,0 +1,321 @@
+package workloads_test
```

Comment:
Similarly to the conversation we've had about testing repositories, are we sure we want to have both an EnvTest-based integration test and a unit test mocking the client? For repositories we have decided to stick to just integration tests, shouldn't we do the same for controllers?

Reply:
For controllers I believe we thought that the unit tests had value because we could test error cases that we couldn't with the integration tests. I'll add this to the team forum this week to discuss further.